CheapShopWeb/Values/SmallerGroups.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System.Collections.Generic;
+
+namespace CheapShopWeb
+{
+    internal class SmallerGroups
```

Comment:
Buvau pakeitęs visas grupes ir smallerGroups į angliškus pavadinimus, tai not sure ar dabar veiks filtravimas

Reply:
Aš nieko nekeičiau, čia tiesiog įdėjau visa folderiuka "Values" Tai jeigu turejai kažkur kitur viskas bus ok. Šita ištrinti reiks, ens tikriausiai tokiu atvėju nenaudojamas